integration-tests/tests/find-workflow.js

```diff
@@ -1,10 +1,14 @@
+const _ = require('lodash/fp')
 const firecloud = require('../utils/firecloud-utils')
 const { withWorkspace } = require('../utils/integration-helpers')
 const { click, clickable, dismissNotifications, findElement, findText, signIntoTerra } = require('../utils/integration-utils')
 const { withUserToken } = require('../utils/terra-sa-utils')
 
 
-const testFindWorkflowFn = withUserToken(withWorkspace(async ({ billingProject, page, testUrl, token, workflowName, workspaceName }) => {
+const testFindWorkflowFn = _.flow(
+  withWorkspace,
+  withUserToken
+)(async ({ billingProject, page, testUrl, token, workflowName, workspaceName }) => {
```

Comment:
What would you think of using `_.flowRight`, or the `_.compose` alias in lodash/fp, to make the function order read more in-order? I think we talked about this before but I don't remember if there was a reason to avoid this.

Reply:
Normally I would use `compose` more when applying wrappers, since I think the order is clearer. But our lodash/fp eslint plugin in the main project enforces the `flowRight` alias, which I think is a terrible name. And even though the tests don't get linted (TOL: maybe they should?), I didn't want to use something here that's not allowed in the main code.

Actually looking at the plugin, it looks like we can override the preferred alias for that lint rule. So perhaps that would be a good follow-on.

Reply:
Sounds good. I agree that this is still more readable than before because it draws a clearer line between composing the functions and invoking the resulting function.